clickhouse-client/src/test/java/com/clickhouse/client/ClickHouseServerForTest.java

```diff
@@ -289,5 +379,44 @@ public static void afterSuite() {
         if (clickhouseContainer != null) {
             clickhouseContainer.stop();
         }
+
+        if (isCloud()) {
+            ClickHouseNode server = ClickHouseNode.builder(ClickHouseNode.builder().addOption(ClickHouseClientOption.SSL.getKey(), "true").build())
```

Comment:
That looks like a lot of logic, should we consider moving it put of the existing class?